src/dataset-builder/DatasetBuilder.ts

```diff
@@ -669,7 +659,10 @@ export const DatasetBuilderView: React.FC<DatasetBuilderProps> = (props) => {
     initialState || homepageState.new()
   );
   const [cohorts, setCohorts] = useState<Cohort[]>([]);
-  const [selectedCohorts, setSelectedCohorts] = useState([] as HeaderAndValues<Cohort>[]);
+  const defaultHeader = 'Saved cohorts';
```

Comment:
What do you think about moving this up so it isn't mixed in with the state initialization?